hnn_core/dipole.py

```diff
@@ -55,6 +55,11 @@ def simulate_dipole(net, tstop, dt=0.025, n_trials=None, record_vsoma=False,
     if n_trials < 1:
         raise ValueError("Invalid number of simulations: %d" % n_trials)
 
+    if not net.connectivity:
```

Comment:
use `len` because `net.connectivity` is list not boolean

Reply:
You've unknowingly stumbled into the most controversial opinion the 21st century :smile: 
https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty
https://stackoverflow.com/questions/6459513/why-implicitly-check-for-emptiness-in-python

Reply:
interesting Python feature :) I think I still prefer the explicit form though but you can decide